spec/mailers/user_mailer_spec.rb

```diff
@@ -131,4 +131,29 @@
       end
     end
   end
+
+  describe "#claim_created_support_notification" do
+    subject(:claim_confirmation_email) { described_class.with(service: support_user.service).claim_created_support_notification(claim) }
+
+    context "when a claim has been submitted" do
```

Comment:
I think the description is wrong here. As per [this comment](https://github.com/DFE-Digital/itt-mentor-services/pull/384#discussion_r1528874713), I think the context is when a claim has been created?

```suggestion
    context "when a claim has been created" do
```